src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java

```diff
@@ -5,91 +5,94 @@
-    WikiBasePageObject base = new WikiBasePageObject();
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.typeInWikiName(cnw1.getWikiName());
-    cnw1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.submitToLogInSignUp();
-    cnwLogin.submitLogin();
-    cnwLogin.verifyEmptyUserNameValidation();
-  }
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_002"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_002_wrongPassword() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
+//    cnwLogin.typeInUserName(credentials.userName);
+//    cnwLogin.typeInPassword(credentials.password + "1");
+//    cnwLogin.submitLogin();
+//    cnwLogin.verifyInvalidPasswordValidation();
+    }
 
-  @Test(groups = {"CNW", "CreateNewWikiLoggedOut_005"})
-  @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
- public void CreateNewWiki_005_invalidUserName() {
-    WikiBasePageObject base = new WikiBasePageObject();
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.typeInWikiName(cnw1.getWikiName());
-    cnw1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.submitToLogInSignUp();
-    cnwLogin.typeInUserName(credentials.userName + "1");
-    cnwLogin.submitLogin();
-    cnwLogin.verifyInvalidUserNameValidation();
-  }
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_003"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_003_blankPassword() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
+//    cnwLogin.typeInUserName(credentials.userName);
+//    cnwLogin.submitLogin();
+//    cnwLogin.verifyBlankPasswordValidation();
+    }
+
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_004"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_004_blankUserName() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+//    cnw1.verifySuccessIcon();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)